snapcraft/tests/test_yaml.py

```diff
@@ -449,6 +449,79 @@ def test_tab_in_yaml(self, mock_loadPlugin):
             'on line 2 of snapcraft.yaml')
 
     @unittest.mock.patch('snapcraft.internal.yaml.Config.load_plugin')
+    def test_yaml_valid_epochs(self, mock_loadPlugin):
+        valid_epochs = [
+            0,
+            '1*',
+            1,
+            '400*',
+            1234,
```

Comment:
how about `'00001'`? Is that valid? I am asking without having looked at how this is spec'ed